main.c

```diff
@@ -554,8 +554,6 @@ main
 
   init_locale();
 
-  umask(077);
```

Comment:
We need to be careful, currently this ensures that all files created by NeoMutt are only accessible for the current user. The default umask on unix systems is 022. When we remove this we could suddenly create temp files, mbox/maildir files, saved mails and so on that are readable by everyone.